src/app/movie-details/movie-details.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { Movies, MoviesGenres } from '../movies.service';
+import { ActivatedRoute } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+
+
+@Component({
+  selector: 'app-movie-details',
+  templateUrl: './movie-details.component.html',
+  styleUrls: ['./movie-details.component.css']
+})
+export class MovieDetailsComponent implements OnInit {
+  imagePath = "https://image.tmdb.org/t/p/w400"
+  public id: any
```

Comment:
nie potrzeba dodawać `public` do zmiennych publicznych

Reply:
Ake z tego co widzę to może być zmienna prywatna. My stosujemy konwencję z `_` do zmiennych/metod prywatnych więc w tym przypadku `private _id: string`

Reply:
jesli wpisuje private _id: string to wyskakuje bład więc zastąpiłem na any i działa